libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -156,6 +158,33 @@ internal class MapboxTripSession(
         }
     private var mapMatcherResult: MapMatcherResult? = null
 
+    private val nativeFallbackVersionsObserver =
+        object : com.mapbox.navigator.FallbackVersionsObserver() {
+            override fun onFallbackVersionsFound(versions: MutableList<String>) {
+                mainJobController.scope.launch {
```

Comment:
I believe this is not needed. NN callbacks run in the same thread as launched (is that right @mskurydin @DmitryAk?) and this is set in the main thread.

Reply:
discussed with @mskurydin some days ago
`onCanReturnToLatest` and `onFallbackVersionsFound ` run on a TileStore's worker thread